code_schemes/disabled.json

```diff
@@ -0,0 +1,206 @@
+{
+  "SchemeID": "Scheme-6e456bff",
+  "Name": "Disabled",
+  "Version": "0.0.0.6",
+  "Codes": [
+    {
+      "CodeID": "code-E-720cdb66",
+      "CodeType": "Meta",
+      "MetaCode": "escalate",
+      "DisplayText": "meta: ESCALATE",
+      "NumericValue": -100080,
+      "StringValue": "escalate",
+      "VisibleInCoda": true,
+      "Shortcut": "e"
+    },
```

Comment:
Do we know what data we expect to get back yet? 
It seems yes/no are the obvious labels to start with, but can understand you wanting to wait.

Reply:
Yup wanted to be sure. We've done the thematic analysis and have updated this with yes/no.